src/StudentOrganizer.Core/TextFile1.txt

```diff
@@ -0,0 +1,3 @@
+﻿Todo
```

Comment:
-- Przykładowy komentarz --
Ten plink to jakieś własne notatki to może dodaj do .gitignore albo wrzuć gdzieś indziej